LandSandBoat/server
# [sql] [lua] Add Pools/Skill Lists/Abilities for Trial Sized Avatars

**_I affirm:_**

## What does this pull request do?

* Adds Mob Pools and Skill Lists for Trial Sized Avatar fights 
* Updates Groups to use the new Mob Pools and Skill Lists 
* Adds Missing Mob Ability scripts for Garuda, Shiva, Ifrit, Ramuh, and Titan.

Closes #2719 

## Steps to test these changes

Follow steps/prerequisites to participate in Trial Sized Trial by (Ice/Fire/Earth/Wind/Water/Thunder)
Observe Mobs will only use abilities from Level 20 and Below
